src/components/home/ScheduleCalendar.module.scss

```
@@ -134,14 +168,62 @@
 }
 
 .calendarHeader {
-  display: flex;
+  display: grid;
+  // 좌우 열을 같은 폭(1fr)으로 잡아 가운데 열이 헤더의 진짜 중앙에 오게 한다.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)